lms/urls.py

```diff
@@ -20,9 +20,7 @@
 from openedx.features.enterprise_support.api import enterprise_enabled
 
 
-# Uncomment the next two lines to enable the admin:
-if settings.DEBUG or settings.FEATURES.get('ENABLE_DJANGO_ADMIN_SITE'):
-    admin.autodiscover()
+django_autodiscover()
```

Comment:
Sorry, I meant literally `django_admin.autodiscover()` or `django_autodiscover()` in both places. Nitty - but avoids any need to think about if they do different things.